hoomd/md/ForceComposite.h

```diff
@@ -232,6 +297,12 @@ class PYBIND11_EXPORT ForceComposite : public MolecularForceCompute
 
     //! Compute the forces and torques on the central particle
     virtual void computeForces(uint64_t timestep);
+
+    /// Number of rigid bodies on the local rank.
+    unsigned int m_n_rigid;
+
```

Comment:
HOOMD prefers this style of doxygen comment:
```suggestion
    unsigned int m_n_rigid; //!< Number of rigid bodies on the local rank.
```
Also, `m_n_rigid` should be explicitly defaulted to 0 in the constructor initialization list.